docs/user/setting_up_your_development_environment.md

```diff
@@ -0,0 +1,78 @@
+# Setting up your development environment
+## Overview
+Garage is a Python package which can be installed in most Python 3.6+
+environments using standard commands, i.e.
+
+``pip install --user garage``
+
+However, to ensure that your environment is consistent with other
```

Comment:
We use these tools because they guarantee that there are no packages interfering with the required packages to run garage. This also ensures that a garage installation does not interfere with other packages installed on a user's machine.